Saudox/resources/views/profissional/criar_paciente.blade.php

```diff
@@ -24,30 +26,40 @@
+                        <label class="required">Nome do Paciente:</label><br>
                         <input value="{{ old('nome_paciente') }}" placeholder="Nome Completo" type="text" name="nome_paciente" id="nome_paciente">
+                        <label class="required">Cpf do Paciente:</label><br>
                         <input value="{{ old('cpf') }}" class="noscroll" onkeypress='validar_apenas_numeros(event)' placeholder="CPF (apenas números)" type="number"  name="cpf" id="cpf">
-                        <h3>Sexo</h3>
+                        <label class="required">Sexo:</label><br>
                         <label for="masculino">Masculino</label>
                         <input value="0" type="radio" name="sexo" id="sexo">
                         <label for="feminino">Feminino</label>
                         <input value="1" type="radio" name="sexo" id="sexo">
+                        <br>
+                        <label class="required">Naturalidade:</label><br>
                         <input value="{{ old('naturalidade') }}" placeholder="Naturalidade" type="text" name="naturalidade" id="naturalidade">
+                        <label class="required">Data de nascimento:</label><br>
                         <input value="{{ old('data_nascimento') }}" class="noscroll" type="date"  name="data_nascimento" id="data_nascimento">
+                        <label class="required">Responsável:</label><br>
                         <input value="{{ old('responsavel') }}" placeholder="Responsável" type="text" name="responsavel" id="responsavel">
+                        <label class="required">Número de irmãos:</label><br>
                         <input value="{{ old('numero_irmaos') }}" class="noscroll" onkeypress='validar_apenas_numeros(event)' placeholder="Número de Irmãos" type="number"  name="numero_irmaos" id="numero_irmaos">
-                        <input value="{{ old('lista_irmaos') }}" placeholder="Nome dos Irmãos" type="text" name="lista_irmaos" id="lista_irmaos">
-                        <h3>Filho Biológico ou Adotivo?</h3>
+                        <label>Nome dos irmãos:</label><br>
+                        <input value="{{ old('lista_irmaos') }}" placeholder="Separado por vírgula" type="text" name="lista_irmaos" id="lista_irmaos">
+                        <label class="required">Filho biológico ou adotivo?</label><br>
                         <label for="biologico">Biológico</label>
                         <input value="0" type="radio" name="tipo_filho_biologico_adotivo" id="tipo_filho_biologico_adotivo">
                         <label for="adotivo">Adotivo</label>
                         <input value="1" class="command_hidden" type="radio" name="tipo_filho_biologico_adotivo" id="tipo_filho_biologico_adotivo">
                         <div class="hidden">
-                            <h3>Filho sabe que é Adotado?</h3>
+                            <label>Filho sabe que é adotado?</label><br>
```

Comment:
## Do mesmo modo que o cadastro do Profissional, realizar uma varredura para identificar onde realmente é necessário o uso do required.
__O campo referente a 'biológico/adotado' por exemplo, é obrigatório__

Reply:
Essa parte ai é do filho SABER se é adotado, ela só aparece se ele for adotado. Não é required

Reply:
## Ótima observação, acabei de ver no controller a implementação que tu fez.
Muito bem